runtime/js-compute-runtime/host_interface/host_api.h

```diff
@@ -13,6 +14,8 @@
 #include "core/allocator.h"
 #include "js/TypeDecls.h"
 
+#include "host_interface/fastly.h"
```

Comment:
@elliottt - I feel like I shouldn't be pulling this in here but couldn't think of anything better to do...

Reply:
I pushed a commit to this branch to address that. Basically, instead of having an explicit constructor, have a private function in host_api.cpp that makes a value of `FastlySendError` from the type that comes from fastly.h. As the only place the error is constructed is in host_api.cpp, it works out fine to have that as a private function in there.